src/controllers/authController.js

```diff
@@ -1,13 +1,41 @@
+const { userService } = require("../services");
+const { keyCheck } = require("../utils/keyCheck");
+
 const socialLogin = async (req, res) => {
-  const token = req.user;
+  const result = req.user;
```

Comment:
```suggestion
  const user = req.user;
```

할당되는 데이터에 비해 `result`는 적절하지 않은 변수명인 것 같습니다.

Reply:
넵!